frontend/src/scenes/feedback/Feedback.tsx

```diff
@@ -0,0 +1,384 @@
+                                            },
+                                        },
+                                    ]}
+                                />
+                                <Form logic={feedbackLogic} formKey={formKey} className="my-4">
+                                    <div className="flex gap-4">
+                                        <div className="flex-1">
+                                            <Field name="name">
+                                                <LemonInput placeholder="Feedback Sent" />
+                                            </Field>
+                                        </div>
+                                        <div className="flex-1">
+                                            <div className="flex gap-4">
+                                                <div className="flex-1">
+                                                    <Field name="properties">
+                                                        <LemonInput placeholder="$feedback,$feedback2" />
+                                                    </Field>
+                                                </div>
+                                                <div>
+                                                    <LemonButton
+                                                        type="primary"
+                                                        onClick={() => {
+                                                            feedbackLogic.actions.addFeedbackProperty()
+                                                        }}
+                                                    >
+                                                        Add
+                                                    </LemonButton>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </Form>
+                            </div>
+                        </div> */}
+                    </div>
+                </div>
+            </ExpandableSection>
+        </div>
+    )
+}
+
+function getFilters(eventName: string): Record<string, any> {
+    return {
+        insight: 'TRENDS',
+        events: [{ id: eventName, name: eventName, type: 'events', order: 0 }],
+        actions: [],
+        display: 'ActionsLineGraph',
+        interval: 'day',
+        new_entity: [],
+        properties: [],
+        filter_test_accounts: false,
+        date_from: '-30d',
+    }
+}
+
+function InAppFeedback({
+    config,
+}: {
+    config: {
+        eventName: string
+        feedbackProperty: string
+    }
+}): JSX.Element {
+    const { eventName } = config
+    const filters = getFilters(eventName)
+
+    const { events } = useValues(feedbackLogic)
+
+    // TODO in next PR call the events endpoint to get the feedback events
```

Comment:
nit: this context gets lost once the PR goes in, best to just say what needs to be done since "next PR" will become ambiguous